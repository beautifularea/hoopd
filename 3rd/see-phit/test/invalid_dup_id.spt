R"*(
<span >
  <p  color="red" height='1' something="test" > 
    Extra whitespace after close and open tags
  </p  >
  
  <DIV id="hello"> 
    Extra space after void tag
  </DIV>
  <IMG />

  <DIV> 
    <DIV id='hello'> 
      Nested tag, void tag
    </DIV>
    <IMG/>
  </DIV>
  
  <P>Inline tag</P>
  <Span>Mixed case tag name</SPAN>

</span>

<DIV>
  <DIV> 
    Multiple top level nodes
  </DIV>
</DIV>
)*"_html; 