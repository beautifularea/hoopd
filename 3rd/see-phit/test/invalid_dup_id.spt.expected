Warning() [with int ROW = 13; int COL = 14; WHAT = spt::Duplicate_ID_on_tag]':
